### PR TITLE
Fix for pickerViews with attributedTitleForRow called

### PR DESCRIPTION
**Pull request by ArielSD**

### CODE REVIEW

**Comment by ArielSD** [comment]
I commit the FAQ change before I signed the CLA. Rerunning now.

File: docs/faq.md [hunk 1]
```diff
@@ -362,6 +362,24 @@ EarlGrey.select(elementWithMatcher:matcher)
                   assertWithMatcher:grey_notNil()];
 ```
 
+**How do I dismiss system alerts, like being asked to allow push notifications?**
+Include a launch argument for testing, that doesn't register your app for notifications.
+
+Something like:
+
+```if [[[NSProcessInfo processInfo] arguments] containsObject:argument] {
```
Comment (sid-github) [inline docs/faq.md]: use newlines before and after ``` and 'objc' specifier, this content is not rendered correctly in markdown:
```objc
// Your code comes here
```

File: docs/faq.md [hunk 2]
```diff
@@ -362,6 +362,24 @@ EarlGrey.select(elementWithMatcher:matcher)
                   assertWithMatcher:grey_notNil()];
 ```
 
+**How do I dismiss system alerts, like being asked to allow push notifications?**
```
Comment (khandpur) [inline docs/faq.md]: Would you mind pulling this FAQ out to its own PR? seems small enough.

File: docs/faq.md (hunk 1 above)
Comment (ArielSD) [inline docs/faq.md]: Going to put this in a separate PR.

File: docs/faq.md (hunk 2 above)
Comment (ArielSD) [inline docs/faq.md]: Sure thing.

**Comment by ArielSD** [comment]
Starting over with two separate branches.
